charts/stable/vikunja/values.yaml

```diff
@@ -160,21 +127,11 @@ service:
   main:
     ports:
       main:
-        port: 10220
+        port: 3456
```

Comment:
Why did we change that?
It is explicitly set to a non-default to test that it wont break if a user changes it.

(Note that vikunja allows changing the port. So a change to use `targetPort` won't be accepted)

Reply:
I made the change thanks !